server/usecases/board/list.go

```diff
@@ -0,0 +1,168 @@
+package board
+
+import (
+	"database/sql"
+
+	"github.com/h3poteto/fascia/lib/modules/database"
+	"github.com/h3poteto/fascia/lib/modules/logging"
+	"github.com/h3poteto/fascia/server/domains/list"
+	"github.com/h3poteto/fascia/server/domains/repo"
+	"github.com/h3poteto/fascia/server/domains/task"
+	repository "github.com/h3poteto/fascia/server/infrastructures/list"
+)
+
+// InjectDB set DB connection from connection pool.
+func InjectDB() *sql.DB {
+	return database.SharedInstance().Connection
+}
+
+func InjectListRepository() list.Repository {
+	return repository.New(InjectDB())
+}
+
+func ListHasCloseAction(list *list.List) (bool, error) {
+	return list.HasCloseAction()
+}
+
+func FindList(projectID, listID int64) (*list.List, error) {
+	return list.Find(projectID, listID, InjectListRepository())
+}
+
+func CreateList(projectID, userID int64, title, color string, optionID sql.NullInt64, isHidden bool) (*list.List, error) {
```

Comment:
__[Golint]__ exported function CreateList should have comment or be unexported ([link](https://sider.review/gh/repos/50192141/pulls/454#issue-3810551))

<sub>posted by Sider</sub>